internal/fullnode/pvc_builder.go

```diff
@@ -6,10 +6,16 @@ import (
 	cosmosv1 "github.com/strangelove-ventures/cosmos-operator/api/v1"
 	"github.com/strangelove-ventures/cosmos-operator/internal/diff"
 	"github.com/strangelove-ventures/cosmos-operator/internal/kube"
+	"gopkg.in/inf.v0"
 	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
+	SnapshotGrowthFactor = 102
```

Comment:
In Go, if the constant begins with a capital letter, it is exported (public). So other packages can use it. In this case, it doesn't appear another package will need it. Can we make it lowercase then? (unexpected)

Fyi, also applies to types, variables, functions and methods.